movie-browser/src/App.js

```diff
@@ -15,12 +19,18 @@ class App extends Component {
   }
 //    fetchMovies('films');
 
+toggleMovieSection = () => {
+  this.setState({ shouldShowAllMovies: !this.state.shouldShowAllMovies });
+};
+
   render() {
+    const { movies } = this.state;
```

Comment:
u can also destructure `shouldShowAllMovies`

Reply:
I don't know what that means

Reply:
it means u can do the same for shouldShowAllMovies that you're doing for `movies`, destructuring it. `const { movies, shouldShowAllMovies } = this.state;`

Reply:
I still don't understand. `movies` is a large array of data, `shouldShowAllMovies` is just a simple toggle. How can you destructure it?

Reply:
Destructure is just doing `{ variable1, variable2 } = someObject`. Nevermind, I will explain on a later stage, disregard this comment.